### ISSUE TITLE
Ticking Entity Crash

### ISSUE THREAD

**Issue by GreatOrator** [BluSunrize/WitchingGadgets#89]
So after a lot of testing I found loading a map with the latest version of WG 1.1.10 crashes to desktop when loading any world (even a new one) with the following log:

http://pastebin.com/PvjU5zKW

I tried disabling transmutation according to an earlier crash report but still get the same crash, when I revert back to 1.1.6 I have no issues.  I am using the latest Traveler's Gear (1.16.6) and forge 1448.

This is in a modpack, but I only get this crash when updating to the newest WG.

**Comment by BluSunrize** [comment]
That crashlog doesn't tell me anything so: No idea.

**Comment by GreatOrator** [comment]
Ya, had concerns on that too since it took me a few hours to even narrow it down.

**Comment by BluSunrize** [comment]
Let me rephrase: Nothing in that log points to WG. It's also not a ticking entity, it's a network crash.

**Comment by GreatOrator** [comment]
I know, posted the title while tired, but trust me, it was frustrating trying to pinpoint (had a ton of mod updates) as I was adding/removing each of the mods one at a time and restarting.  With the the update it crashed each time no matter what mod I removed or added.  It was only upon completely removing (going back to 1.1.6) that everything loaded properly and this was after three hours of testing.  I honestly wish I had more to give you to go on.

**Comment by GreatOrator** [comment]
Something came up that made me think to check with yu as well...does any part of this mod rely on the botania api?

**Comment by BluSunrize** [comment]
Yes, but that should not be involved here. And I only use a very small part of it.

**Comment by GreatOrator** [comment]
Ok, going to do further testing then.  I recently found another mod that was crashing and it was due to an api change in botania.  In my pack I had not updated botania until the latest update (decay was not preferred) and now that is no longer an issue I was able to update that mod as well as botania.  Might be what was causing the crash with your mod as well.  Will let you know.

**Comment by GreatOrator** [comment]
Nope, still same crash after update...this has me stumped
